content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -12,39 +12,94 @@ a single controllable composite application.
 
 For applications up to a certain size, all of the microservices comprising the
 application can be running on a single orchestration platform (e.g., Kubernetes cluster).
-However, for many reasons such as scale, redundancy, compliance, etc., most applications
-will eventually need to be distributed and have some of their services running elsewhere.
+Eventually, you will need to distribute most applications and run some or all of their services elsewhere due to one or more of the following reasons:
```

Comment:
you may need to

Reply:
fixed in https://github.com/istio/istio.io/pull/4725/commits/773418edeb80c4779cfacc6df551cb540119560d